libpod/container_inspect.go

```diff
@@ -703,12 +703,14 @@ func (c *Container) generateInspectContainerHostConfig(ctrSpec *spec.Spec, named
 	}
 	hostConfig.CapAdd = capAdd
 	hostConfig.CapDrop = capDrop
-
-	// IPC Namespace mode
-	ipcMode := ""
-	if c.config.IPCNsCtr != "" {
-		ipcMode = fmt.Sprintf("container:%s", c.config.IPCNsCtr)
-	} else if ctrSpec.Linux != nil {
+	switch {
+	case !c.config.ShmShare:
+		hostConfig.IpcMode = "private"
```

Comment:
Won't this be true for `ipc=container:` as well?

Reply:
ShmShare would be set for them.